spacy_experimental/coref/coref_component.py

```diff
@@ -242,7 +281,7 @@ def update(
                 continue
             preds, backprop = self.model.begin_update([eg.predicted])
             score_matrix, mention_idx = preds
-            loss, d_scores = self.get_loss([eg], score_matrix, mention_idx)
+            loss, d_scores = self.get_loss([eg], score_matrix[0], mention_idx[0])
```

Comment:
Its `[0]` because now we have the batch dimension and predict works on batches of 1?

Reply:
No, it's because some of the dimensions have changed with processing inputs/outputs, and there's basically an extra dimension.